js/siteorigin-panels/view/builder.js

```diff
@@ -475,10 +476,8 @@ module.exports = Backbone.View.extend( {
         }
 
         // Create the live editor and set the builder to this.
-        this.liveEditor = new panels.view.liveEditor();
-        this.liveEditor.setPostId(postId);
-
-        this.liveEditor.builder = this;
+        this.liveEditor = new panels.view.liveEditor( { builder: this } );
+        this.liveEditor.setPostId( postId );
```

Comment:
I don't see the `setPostId` being called anywhere after this initial call so why not just have `postId` as an option in the constructor and set it in the `initialize` method?
E.g. `this.liveEditor = new panels.view.liveEditor( { builder: this, postId:postId } );`